frontend/src/lib/WebsocketConnection.tsx

```diff
@@ -106,13 +107,19 @@ interface Args {
    * Function to get the auth token set by the host of this app (if in a
    * relevant deployment scenario).
    */
-  getHostAuthToken: () => string | undefined
+  claimHostAuthToken: () => Promise<string | undefined>
+
+  /**
+   * Function to tell the withHostCommunication hoc that we've received
+   * the auth token set by the host of this app.
+   */
+  resetHostAuthToken: () => void
```

Comment:
"reset" feels like the wrong verb for "indicate that we've received a thing". (Or maybe it's the docstring that's wrong?)

Reply:
I think the docstring is probably what needs to be rewritten here since the reset function is where the reset happens